src/main/kotlin/lotto/controller/LottoController.kt

```diff
@@ -0,0 +1,42 @@
+package lotto.controller
+
+import lotto.model.Lotto
+import lotto.model.LottoNumber
+import lotto.model.LottoNumberListGenerator
+import lotto.model.Lottos
+import lotto.model.Price
+import lotto.model.WinNumber
+import lotto.view.InputView
+import lotto.view.OutputView
+
+object LottoController {
+    private val inputView = InputView()
+    private val outputView = OutputView()
+
+    fun runLottoGame() {
+        val price = inputPrice()
+        val purchasedLottoList = generateLottos(price)
+        printResult(price, purchasedLottoList)
+    }
+
+    private fun inputPrice(): Price {
+        val price = Price(inputView.takePurchasedPrice())
+        outputView.resultLottoCount(price.lottoCount)
+        return price
+    }
+
+    private fun generateLottos(price: Price): List<Lotto> {
+        val purchasedLotto = LottoNumberListGenerator(price).generateLottoList(LottoNumber.getLottoNumberRange().shuffled())
```

Comment:
외부에서 랜덤 값 주입 👍 💯 🥇

Reply:
```suggestion
        val purchasedLotto = LottoNumberListGenerator(price)
            .generateLottoList(
                LottoNumber
                    .getLottoNumberRange()
                    .shuffled()
            )
```
함수 체이닝 컨벤션을 적용하면 좋을 것 같아요